tests/cypress/support/dummy-data.js

```diff
@@ -89,8 +90,68 @@ const dummyAWSBucket = {
     ],
 };
 
+function dummyTaskSpec({
```

Comment:
Technically speaking this is not exactly dummy task spec. 
Speaking about cloud storages, they are just response-mocked, that is why they are dummy.
Hovewer we create a task from this specification, so, maybe to place it in another file

Reply:
Moved to new file `default-specs.js`, renamed to `defaultTaskSpec`